js/views/itemEditVw.js

```diff
@@ -217,19 +220,33 @@ module.exports = Backbone.View.extend({
   },
 
   deleteImage: function(e) {
-    var imgIndex = $(e.target).closest('.itemImg').data('index');
+    var imageUploadArray, imgIndex = $(e.target).closest('.itemImg').data('index');
```

Comment:
It's a good practice, but we didn't start with it so we'll have to consider how much of an impact going back to refactor things would have before adding it in across the project. I think it would be fine to add as things are worked on.